955-E.180/T-SP-E.180-2010-MSW-E.adoc

```diff
@@ -49,28 +48,25 @@ E-mail: mailto:tsbtson@itu.int[]
 
 == {{ file.metadata.title[lang] }}
 
-{% assign ordered_sequence = file.data | map: "country_or_area" | sort: lang | uniq %}
-
-
 [cols="3",options="unnumbered,header"]
 |===
 | {{ file.metadata.locale.country_or_area[lang] }}
 | {{ file.metadata.locale.frequency_in_hz[lang] }}
 | {{ file.metadata.locale.cadence_in_s[lang] }}
 
 
-{% for next_country in ordered_sequence %}
-    3+| *{{ next_country[lang] }}*
-    {% for country_data in file.data %}
-        {% if  next_country[lang] == country_data.country_or_area[lang] -%}
-            | {{ country_data.tone_desc[lang] }}
-            {%- if country_data.notes -%}
-            footnote:[{{ country_data.notes[lang] }}]
-            {%- endif -%}
-            | {{ country_data.frequency_in_hz }}
-            | {{ country_data.cadence_in_s }}
-        {% endif %}
-    {% endfor %}
+{% for country_data in file.data %}
+    {% assign next_country = country_data.country_or_area[lang] %}
+    {% if next_country != current_country -%}
+        {% assign current_country = next_country %}
+        3+| *{{ current_country }}*
```

Comment:
Hey @ronaldtse, yeah I was aware of that but, given that the output was correct despite of the indentation, I decided to leave it as it was.
Ok, I have removed the indentation.
Thanks!

Reply:
Thanks @manuel489 ! I was concerned about this because AsciiDoc takes an indented text after a new line as a new verbatim/quote block, e.g.

```
* This is A

  This is supposed to be part of A, but is actually considered as a new and separate verbatim block.
```